cmd/system-probe/main_common.go

```diff
@@ -0,0 +1,97 @@
+package main
```

Comment:
I suggest naming this file `debug_cmd` since `main_common` suggests this is run as part of the main command.